RadixWallet/Features/AccountRecoveryScan/Children/AccountRecoveryScanInProgress/AccountRecoveryScanInProgress+View.swift

```diff
@@ -65,28 +65,14 @@ public extension AccountRecoveryScanInProgress {
 		public var body: some SwiftUI.View {
 			WithViewStore(store, observe: \.viewState, send: { .view($0) }) { viewStore in
 				VStack(alignment: .center, spacing: .medium1) {
-					Text(viewStore.title)
-						.textStyle(.sheetTitle)
-
-					fixedFrameHeader(with: viewStore)
-
-					if viewStore.active.isEmpty {
-						if !viewStore.isScanInProgress {
-							NoContentView("None found.") // FIXME: Strings
-						}
+					if viewStore.isScanInProgress {
+						scanInProgressView(with: viewStore)
 					} else {
-						ScrollView {
-							VStack(alignment: .leading, spacing: .small3) {
-								ForEach(viewStore.active) { account in
-									SmallAccountCard(account: account)
-										.cornerRadius(.small1)
-								}
-							}
-						}
+						scanCompleteView(with: viewStore)
 					}
-
-					Spacer(minLength: 0)
 				}
+				.textStyle(.body1Regular)
+				.foregroundColor(.app.gray1)
 				.controlState(viewStore.loadingState)
 				.presentsLoadingViewOverlay()
 				.padding()
```

Comment:
The horizontal part of this padding should go inside the scrollview, on the VStack, so that the vertical scrollbar appears in the correct place.

Reply:
good catch!